README.md

```diff
@@ -106,7 +106,10 @@ tree -L 1 ~/code/exercism
 
 From within the ruby directory, run the following command:
 
-    bin/generate <slug>
+    bin/generate --update <slug> 
+
+To regenerate tests without the updated canonical data, run the command
+without the `--update` option.
```

Comment:
This sentence is misleading.

The `--update` flag causes the BookKeeping version number to increment.

How about something like: 

"Leaving out the `--update` option will cause the BookKeeping version number to remain the same.
This can be useful when testing generators."